README.md

```diff
@@ -53,7 +53,16 @@ Integration and unit tests are provided.
 
 ## Changelog
 
-**v3.0.27 - TBD**
+**v3.0.27 - 2016 Feb 17**
+
+   + Make sure HTTP proxy authentication works with the @ character in a password.
+   + Make sure sudo authentication test works with Shotgun versions after 6.3.10
+   + Smarter uploading of thumbnails and filmstrips with the upload() method.
+   + Improve Travis build integration of the Python-API to run the full suite of
```

Comment:
I don't think this is really necessary as a release note since it's just for us internally right?